main.nf

```diff
@@ -89,12 +89,14 @@ def helpMessage() {
                                           Requires that --ascat_ploidy is set
       --cf_coeff                    [str] Control-FREEC coefficientOfVariation
                                           Default: ${params.cf_coeff}
+      --cf_contamination_adjustment[bool] Control-FREEC contaminationAdjustment
```

Comment:
```suggestion
      --cf_contamination_adjustment [bool] Control-FREEC contaminationAdjustment
```

Reply:
No space 😱

Reply:
Won't align if I add the space, and as I'll sync with TEMPLATE after, won't really matter anyway...

Reply:
I know. Real world problems! I was just being pedantic :)